### PR TITLE
helm - add support for repo location when running helm diff

### LINKED ISSUE #174: helm diff not working when using kubernetes.core.helm with on-the-fly templating
##### SUMMARY
When using the module _kubernetes.core.k8s_ with on-the-fly templating helm diff seems not to work.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
- kubernetes.core.helm

##### ANSIBLE VERSION
```paste below
ansible 2.10.11
  config file = None
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.9/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.9.6 (default, Jun 29 2021, 19:27:32) [GCC 8.3.0]
```

##### COLLECTION VERSION
```paste below
kubernetes.core    2.1.1
```

##### CONFIGURATION
```paste below
ANSIBLE_FORCE_COLOR(env: ANSIBLE_FORCE_COLOR) = True
DEFAULT_LOAD_CALLBACK_PLUGINS(env: ANSIBLE_LOAD_CALLBACK_PLUGINS) = True
DEFAULT_STDOUT_CALLBACK(env: ANSIBLE_STDOUT_CALLBACK) = yaml
HOST_KEY_CHECKING(env: ANSIBLE_HOST_KEY_CHECKING) = False
```

##### OS / ENVIRONMENT
debian buster

##### STEPS TO REPRODUCE

1. Install helm diff
2. Run this task on a target machine with a valid *values_redis.yml.j2*

```yaml
- name: Install/Upgrade redis
  kubernetes.core.helm:
    chart_repo_url: https://charts.bitnami.com/bitnami
    chart_ref: redis
    release_namespace: '{{ redis_namespace }}'
    name: redis
    chart_version: '{{ redis_helm_version }}'
    release_values: '{{ lookup("template", "values_redis.yml.j2") | from_yaml }}'
```

##### EXPECTED RESULTS
Chart gets deployed without errors

##### ACTUAL RESULTS
I get the following warning even when helm diff is installed

```paste below
[WARNING]: The default idempotency check can fail to report changes in certain cases. Install helm diff for better results.
```

### PR DESCRIPTION
**Pull request by abikouo**
##### SUMMARY

closes #174 

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

``helm``

### CODE REVIEW

File: plugins/modules/helm.py [hunk 1]
```diff
@@ -743,13 +753,14 @@ def main():
...
                     replace,
+                    chart_repo_url,
                 )
                 if would_change and module._diff:
                     opt_result["diff"] = {"prepared": prepared}
             else:
                 module.warn(
                     "The default idempotency check can fail to report changes in certain cases. "
-                    "Install helm diff for better results."
+                    "Install helm diff >= 3.4.1 for better results."
```
Comment (Akasurde) [inline plugins/modules/helm.py:763]: Can you please add a note in the docs section about this? So that anyone who reads the docs will get to know.

Comment (abikouo) [reply plugins/modules/helm.py:763]: yes I will do it.
Thanks

Comment (Akasurde) [reply plugins/modules/helm.py:763]: Thanks,

**Review by gravesm — APPROVED** [review]
Thanks!

**Comment by abikouo** [comment]
regate

**Comment by abikouo** [comment]
regate

**Pull request merged**